app/models/region.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: regions
+#
+#  id                                         :uuid             not null, primary key
+#  claims_funding_available_per_hour_currency :string           default("GBP"), not null
+#  claims_funding_available_per_hour_pence    :integer          default(0), not null
+#  name                                       :string           not null
+#  created_at                                 :datetime         not null
+#  updated_at                                 :datetime         not null
+#
+class Region < ApplicationRecord
+  monetize :claims_funding_available_per_hour_pence, as: :claims_funding_available_per_hour
```

Comment:
Since this is the default name, _this_ should just work:

```suggestion
  monetize :claims_funding_available_per_hour_pence
```